client/src/components/ActivityList/ActivityList.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import "./ActivityList.css";
+
+class ActivityList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      classList: "ActivityList"
+    };
```

Comment:
Can you remove the `classList` from state and turn this into a function component